examples/go-tcx-counter/container-deployment/Containerfile.go-tcx-counter

```
@@ -0,0 +1,30 @@
+ARG BUILDPLATFORM=linux/amd64
+
+FROM --platform=$BUILDPLATFORM golang:1.22 AS go-tcx-counter-build
+
+ARG BUILDPLATFORM
+
+# The following ARGs are set internally by docker or podman on multiarch builds
+ARG TARGETPLATFORM=linux/amd64
+
+RUN apt-get update && apt-get install -y \
+    clang \
+    gcc-multilib \
+    libbpf-dev
+
+WORKDIR /usr/src/bpfman/
+COPY ./ /usr/src/bpfman/
+
+WORKDIR /usr/src/bpfman/examples/go-tcx-counter
+
+# Compile go-tc-counter
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build
+
+
+FROM --platform=$TARGETPLATFORM registry.fedoraproject.org/fedora-minimal:latest
```

```suggestion
FROM registry.fedoraproject.org/fedora-minimal:latest
```
Removed in all the files in the multi-arch PR.

---

Done.